docs/source/architecture/journal.rst

```diff
@@ -310,8 +310,10 @@ necessary transaction processors must be running.
 If any transactions in the genesis block set or change settings, Sawtooth
 requires the `Sawtooth Settings transaction processor <../cli/settings-tp>`_
 or an equivalent implementation.
-For example, if the genesis block configures PoET consensus, this transaction
-processor is handles the transactions with PoET settings.
+For example, the genesis block specifies the :term:`consensus engine` and
+related settings (unless :term:`Devmode consensus` is used), so the Settings
```

Comment:
Why "unless Devmode consensus is used"?

Reply:
I wondered if there's any DevMode-specific settings, and it seems there are:
```
sawtooth.consensus.max_wait_time
    Maximum devmode consensus wait time, in seconds
sawtooth.consensus.min_wait_time
    Minimum devmode consensus wait time, in seconds
```
Unfotunately, they don't have `devmode` in the name to make it obvious they are DevMode-specific

Reply:
Thanks, @danintel - I'll add these settings to the Devmode section.

Reply:
@danintel - Your wonderful FAQ also lists this setting for Devmode:

`sawtooth.consensus.valid_block_publishers`
    `List of public keys for allowed block publishers. For devmode`

@ltseeley - Do you know if this setting is used in Sawtooth 1.2?

Reply:
It's not used in 1.1 or 1.2; the validator code it's used in is actually dead code. It's not in the Devmode engine either, so it looks like that's deprecated.